dpctl/__init__.py

```diff
@@ -1,5 +1,4 @@
-# ===---------------- __init__.py - dpctl module -------*- Cython -*--------===#
-#
+# ===-----------------------------------------------------------------------===#
 #                      Data Parallel Control (dpCtl)
```

Comment:
This line is also need to support in the future.
I would like to have only license at the header and minimum decorations.